how-to/security/install-openvpn.md

```diff
@@ -120,12 +129,13 @@ ca ca.crt
 cert myservername.crt
 key myservername.key
 dh dh.pem
+tls-auth ./easy-rsa/ta.key 0
 ```
 
 Complete this set with a TLS Authentication (TA) key in `etc/openvpn` for `tls-auth` like this:
```

Comment:
I think this directory is missing a `/`, i.e., `/etc/openvpn` (not your change, but since we are here...)